log.go

```diff
@@ -2,11 +2,45 @@ package wrapplog
 
 import (
 	"os"
+	"path"
 
 	"github.com/Sirupsen/logrus"
 )
 
+type WrappHook struct {
+	ServiceName string
+}
+
+func (w *WrappHook) Levels() []logrus.Level {
+	return logrus.AllLevels
+}
+
+func (w *WrappHook) Fire(entry *logrus.Entry) error {
+	entry.Data["service"] = w.ServiceName
+
+	return nil
+}
+
 func init() {
-	logrus.SetFormatter(&logrus.JSONFormatter{})
+	serviceName := os.Getenv("SERVICE_NAME")
```

Comment:
I agree with you, it much better. However, since WEP-007 says:
"...If the environment variable is not set, the field must be set to something descriptive..", the executable name was added as a fallback.